openssl/src/ssl/mod.rs

```diff
@@ -1117,8 +1119,7 @@ make_LibSslError! {
 /// A stream wrapper which handles SSL encryption for an underlying stream.
 pub struct SslStream<S> {
     ssl: Ssl,
-    _method: Arc<ffi::BIO_METHOD>, // NOTE: this *must* be after the Ssl field so things drop right
-    _p: PhantomData<S>,
```

Comment:
If we keep the `S: Send` qualification on `BioMethod` instead, we can remove the `unsafe impl Send` on `SslStream` since it will automatically be `Send` due to `Ssl` and `BioMethod` both being `Send`. Send via composition rather than `unsafe impl` feels safer.